LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Global Public Inclusive Infrastructure
```

Comment:
I believe this is not a body to which copyright can legally be assigned, and that you should instead use RTF-I - however, IANAL